greengrass_ipc/source/GreengrassCoreIpcModel.cpp

```diff
@@ -2433,6 +3036,105 @@ namespace Aws
             AbstractShapeBase::s_customDeleter(static_cast<AbstractShapeBase *>(shape));
         }
 
+        void SubscribeToCertificateUpdatesResponse::SerializeToJsonObject(Aws::Crt::JsonObject &payloadObject) const
```

Comment:
Resolving this comment - my guess is that this is to keep compatibility with the model. While it is something to note, it can be left. The same for the other comment where I mentioned the same thing.